examples/recsys2020.ipynb

```diff
@@ -407,28 +624,35 @@
     "time_preproc = time.time()-time_preproc_start"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can take a look in the output folder."
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": 12,
+   "execution_count": 23,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "_metadata\t part.2.parquet   part.31.parquet  part.43.parquet\n",
-      "part.0.parquet\t part.20.parquet  part.32.parquet  part.44.parquet\n",
-      "part.1.parquet\t part.21.parquet  part.33.parquet  part.45.parquet\n",
-      "part.10.parquet  part.22.parquet  part.34.parquet  part.46.parquet\n",
-      "part.11.parquet  part.23.parquet  part.35.parquet  part.47.parquet\n",
-      "part.12.parquet  part.24.parquet  part.36.parquet  part.48.parquet\n",
-      "part.13.parquet  part.25.parquet  part.37.parquet  part.49.parquet\n",
-      "part.14.parquet  part.26.parquet  part.38.parquet  part.5.parquet\n",
-      "part.15.parquet  part.27.parquet  part.39.parquet  part.50.parquet\n",
-      "part.16.parquet  part.28.parquet  part.4.parquet   part.6.parquet\n",
-      "part.17.parquet  part.29.parquet  part.40.parquet  part.7.parquet\n",
-      "part.18.parquet  part.3.parquet   part.41.parquet  part.8.parquet\n",
-      "part.19.parquet  part.30.parquet  part.42.parquet  part.9.parquet\n"
+      "_metadata\t part.2.parquet   part.31.parquet  part.43.parquet\r\n",
+      "part.0.parquet\t part.20.parquet  part.32.parquet  part.44.parquet\r\n",
+      "part.1.parquet\t part.21.parquet  part.33.parquet  part.45.parquet\r\n",
+      "part.10.parquet  part.22.parquet  part.34.parquet  part.46.parquet\r\n",
+      "part.11.parquet  part.23.parquet  part.35.parquet  part.47.parquet\r\n",
+      "part.12.parquet  part.24.parquet  part.36.parquet  part.48.parquet\r\n",
+      "part.13.parquet  part.25.parquet  part.37.parquet  part.5.parquet\r\n",
+      "part.14.parquet  part.26.parquet  part.38.parquet  part.6.parquet\r\n",
+      "part.15.parquet  part.27.parquet  part.39.parquet  part.7.parquet\r\n",
+      "part.16.parquet  part.28.parquet  part.4.parquet   part.8.parquet\r\n",
+      "part.17.parquet  part.29.parquet  part.40.parquet  part.9.parquet\r\n",
+      "part.18.parquet  part.3.parquet   part.41.parquet\r\n",
+      "part.19.parquet  part.30.parquet  part.42.parquet\r\n"
```

Comment:
did you manage to get this working on windows ? (asking about the \r\n fields at the end there ). Doesn't require any code changes here, just wondering =)

Reply:
I havent tried NVTabular on windows yet. I am sure, if this should be different on windows? The associated output, which has '\r\n' is produced by the cell !ls $BASE_DIR/preprocess. I just list the output files to see what is produced. I think running the notebook on Windows may produce it with '\n'